spec/spec_helper.rb

```diff
@@ -32,6 +32,7 @@ def load(file, wrap = false)
 if !ENV['COSMOS_NO_SIMPLECOV']
   require 'simplecov'
   if ENV['GITHUB_WORKFLOW']
+    ENV['CODECOV_TOKEN'] = "a9684b83-9a5d-4a27-b6cd-35e445b5fb0a"
```

Comment:
Is this a secret that is ok to checkin?

Reply:
Not really ... unless someone else wants to post coverage results to our CodeCov. I'm actually not even sure it's necessary since it was working before. I'll remove and see what happens.